dynamallow/model.py

```diff
@@ -132,6 +119,30 @@ class Schema:
                 id = fields.String(required=True)
                 name = fields.String()
                 color = fields.String(validate=validate.OneOf(('purple', 'red', 'yellow')))
+                compound = fields.Dict(required=True)
+
+            def say_hello(self):
+                print("Hello.  {name} here.  My ID is {id} and I'm colored {color}".format(
+                    id=self.id,
+                    name=self.name,
+                    color=self.color
+                ))
+
+        # Schematics example
```

Comment:
Let's end the previous restructuredtext `.. code-block::` and start a new one here so both examples are in separate `<pre>` areas.